authentication/serializers.py

```diff
@@ -344,3 +345,22 @@ class RegisterExtraDetailsSerializer(SocialAuthSerializer):
     def create(self, validated_data):
         """Try to 'save' the request"""
         return super()._authenticate(SocialAuthState.FLOW_REGISTER)
+
+
+class CustomSendEmailResetSerializer(SendEmailResetSerializer):
+
+    def get_user(self, is_active=True):
```

Comment:
Normally when some library functionality is overridden, we would call `super()` somewhere to default to the base implementation, or we would completely replace the logic. This is directly copying most of the logic of the overridden method, so I think it would be useful to note that explicitly 

```suggestion
    # NOTE: This directly copies the implementation of djoser.serializers.UserFunctionsMixin.get_user
    # and only changes the User query. If this method is changed in an updated Djoser
    # release, this method may need to be updated as well.
    def get_user(self, is_active=True):
```